solution-base/build.sh

```diff
@@ -54,15 +54,15 @@ MONGODB_NAMESPACE=${SOLUTION_ENV}
 MONGODB_REGISTRY=${SOLUTION_REGISTRY}
 MONGODB_IMAGE_NAME="mongodb"
 MONGODB_IMAGE_TAG=$(yq eval ".mongodb.tag" $SOLUTION_BASE_DIR/deps.yaml)
-MONGODB_INIT_IMAGE_NAME="bitnami-shell"
+MONGODB_INIT_IMAGE_NAME="os-shell"
 MONGODB_INIT_IMAGE_TAG=$(yq eval ".mongodb-mongodb-shell.tag" $SOLUTION_BASE_DIR/deps.yaml)
 MONGODB_EXPORTER_IMAGE_NAME="mongodb-exporter"
 MONGODB_EXPORTER_IMAGE_TAG=$(yq eval ".mongodb-exporter.tag" $SOLUTION_BASE_DIR/deps.yaml)
 MONGODB_SHARDED_IMAGE_NAME="mongodb-sharded"
 MONGODB_SHARDED_IMAGE_TAG=$(yq eval ".mongodb-sharded.tag" $SOLUTION_BASE_DIR/deps.yaml)
 MONGODB_SHARDED_EXPORTER_IMAGE_NAME="mongodb-exporter"
 MONGODB_SHARDED_EXPORTER_IMAGE_TAG=$(yq eval ".mongodb-sharded-exporter.tag" $SOLUTION_BASE_DIR/deps.yaml)
-MONGODB_SHARDED_SHELL_IMAGE_NAME="bitnami-shell"
+MONGODB_SHARDED_SHELL_IMAGE_NAME="os-shell"
```

Comment:
should be retrieved from `deps.yaml`, like the tag

Reply:
done here : https://github.com/scality/Zenko/pull/2080/commits/a4e595610f4704d8e6a6a044615877e93af857f1#diff-527481e4f2f813ceb9a952c58b56268535ce2bc9fe197644fbbf33f00ca37b79R57